docs/exercises/5-multi_region.rst

```diff
@@ -348,14 +354,15 @@ in us-west-2.
 
     .. group-tab:: Java
 
+
         We have built a simple bash script that sets the grant, thereby disabling the use of the CMK in us-east-2.
         Run the script as below.
 
         Note, be sure to save the grant_id that outputs to the CLI. You will need this to revoke the grant.
```

Comment:
"Note, be sure to save the grant_id that outputs to the CLI. You will need this to revoke the grant.
Write"
They don't need to save this anymore the updated bash scripts write and pull the grant id from a txt file

Reply:
Per offline discussion, fixing this in the code update pass